engine/src/flutter/lib/web_ui/lib/src/engine/arena.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+abstract class Collectable {
```

Comment:
nit: Flutter codebase prefers American English (e.g. "color" instead of "colour"), so this should probably be "Collectible".

Reply:
`Collectable` is fine American English. 😄